src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ActionButtons = ({ actionButtons }) => {
+  if (actionButtons) {
+    return (
+      <React.Fragment>
+        <a href="#!">Edit</a> <a href="#!">Delete</a>
```

Comment:
When you put #! on links normally they go to some page and I think since they are not yet connected let them be un clickable

Reply:
The Links are currently dummy links, and when hovered upon there's a visual cue that tells the user that these links are not available at the moment. Even if a user decides to click on any of the links it will not navigate the user elsewhere. Moreover, the functionality of those links will be worked on shortly.

Reply:
Fixed, thank you for your feedback.